src/logic/basic.lean

```diff
@@ -508,6 +508,18 @@ or.comm.trans decidable.or_iff_not_imp_left
 
 theorem or_iff_not_imp_right : a ∨ b ↔ (¬ b → a) := decidable.or_iff_not_imp_right
 
+-- See Note [decidable namespace]
+protected lemma decidable.not_or_iff_imp [decidable a] : ¬ a ∨ b ↔ (a → b) :=
```

Comment:
Do we really need this given we have decidable.imp_iff_not_or?

Reply:
It's practical in the sense that arguably neither is simpler than the other and going between them requires rewriting by `not_not`. It also completes a square:
* ` decidable.imp_iff_not_or` corresponds to `or.resolve_left`
* ` decidable.not_or_iff_imp` corresponds to `or.neg_resolve_left`

Reply:
Sorry, my claim is that `iff.symm` translates between the two. Am I mistaken?

Reply:
Seems like I'm wrong. Most places use `imp_iff_not_or`, not `←imp_iff_or_not`. I've turned the maps around.